episodes/02-image-basics.md

```diff
@@ -8,11 +8,11 @@ exercises: 5
 
 - Define the terms bit, byte, kilobyte, megabyte, etc.
 - Explain how a digital image is composed of pixels.
-- Recommend using imageio (resp. skimage) for I/O (resp. image processing) tasks.
+- Recommend using imageio (resp. scikit-image) for I/O (resp. image processing) tasks.
 - Explain how images are stored in NumPy arrays.
 - Explain the left-hand coordinate system used in digital images.
 - Explain the RGB additive colour model used in digital images.
-- Explain the order of the three colour values in skimage images.
+- Explain the order of the three colour values in scikit-image images.
```

Comment:
Hmm, I'm not sure what 'scikit-image images' are... For now, in scikit-images, images are NumPy arrays (as taught in this lesson). Maybe the above was meant to be 'digital images?'